README.md

```diff
@@ -1,11 +1,121 @@
-# akka
+# Akka
 
+This project contains a sample Akka application, along with configurations for local execution and deployment to various platforms.
+
+## Local Development & Execution
+
+### Prerequisites
+- Docker installed
+
+### Build & Run Locally
+1.  Build the container image:
+    ```bash
+    docker build -t akka-app .
+    ```
+2.  Run the application:
+    ```bash
+    docker run -p 5000:5000 akka-app
+    ```
+3.  Access the application at `http://127.0.0.1:5000`.
+
+## AWS ECS Fargate Deployment (via Terraform)
+
+This setup deploys the Akka application to AWS ECS Fargate, making it accessible via its private IP address within the configured VPC.
+
+### Prerequisites
+- AWS Account & IAM User with necessary permissions to create ECS, IAM, EC2 (Security Groups), CloudWatch Logs resources.
+- AWS CLI configured with credentials (e.g., via `aws configure` or environment variables).
+- Terraform installed.
+- Docker installed (for pushing the image to a registry if not using a public one like `images.paas.redhat.com/rhel-ai-cicd/akka:latest`).
+
+### Deployment Steps
+
+1.  **Navigate to the deployment directory:**
+    ```bash
+    cd deployment
+    ```
+
+2.  **Update Image (If Necessary):**
+    The current configuration in `deployment/main.tf` uses `image = "images.paas.redhat.com/rhel-ai-cicd/akka:latest"`.
```

Comment:
To align with open source best practices, the container image should ideally be published to a public registry such as GitHub Container Registry or another accessible platform. Not a blocker for this PR. Just something to keep in mind for a future update.

Reply:
I've updated it to use the new ghcr.io image.